basepluscommission.java

```diff
@@ -0,0 +1,111 @@
+package DB;
+
+import java.sql.*;
+
+public class BasePlusCommission {
+
+    public static void show() {
+        {
```

Comment:
The function already had an opened curly bracket. so this bracket is unnecessary.